docs/client/connecting-kn-to-your-cluster.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Connecting the `kn` Client to your cluster"
```

Comment:
I commented about this on slack, but since I had to delete the /client/ index file and readme to make things look okay for the last release, they should be added back in this PR.